js/components/common/react_modal.js.jsx

```diff
@@ -15,6 +15,9 @@ import {
 } from "react-leaflet"
 import L from "leaflet"
 import { getCourse, getPost } from "../common/utils"
+import { AgGridReact } from "ag-grid-react"
+
+const convertToDay = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday"]
```

Comment:
It's better to use an object here (for constant-time lookup). Also "convert" is a verb, so a better name for the object would be something like `DAY_TO_INT` (also using the naming convention of ALL_CAPS for constants).